src/openvic-simulation/map/MapDefinition.cpp

```diff
@@ -12,6 +14,8 @@ using namespace OpenVic::NodeTools;
 
 MapDefinition::MapDefinition() : dims { 0, 0 }, max_provinces { ProvinceDefinition::MAX_INDEX } {}
 
+RiverSegment::RiverSegment(uint8_t new_size, std::vector<ivec2_t> new_points) : size { new_size }, points { std::move(new_points) } {}
```

Comment:
```suggestion
RiverSegment::RiverSegment(uint8_t new_size, std::vector<ivec2_t>&& new_points)
	: size { new_size }, points { std::move(new_points) } {}
```
Make `new_points` an r-value reference and move the initialiser list to the next line